src/query/remote/compressed_codecs.go

```diff
@@ -151,7 +151,7 @@ SeriesIterator also has a TagIterator representing the tags associated with it.
 This function transforms a SeriesIterator into a protobuf representation to be
 able to send it across the wire without needing to expand the series.
 */
-func compressedSeriesFromSeriesIterator(
+func CompressedSeriesFromSeriesIterator(
```

Comment:
Wi need to udate comment here too

Reply:
This comment needs to start with `// CompressedSeriesFromSeriesIterator ...` now since it's an exposed type.

Also time to get rid of the `/*` and `*/` and use typical `//` for `godoc` to process it.